posts/2024/2024_06_28_Inigo_week_5.rst

```diff
@@ -0,0 +1,31 @@
+Week 5 into GSoC 2024: Vacation, starting with the conditional AutoEncoder
+==========================================================================
+
+
+.. post:: June 28 2024
+    :author: Iñigo Tellaetxe
+    :tags: google
+    :category: gsoc
+
+
+
+What I did this week
+~~~~~~~~~~~~~~~~~~~~
+
+Hi everyone! This week I have been on vacation, so I have not been able to work on the project as much as the previous weeks. However, I have been thinking about the next steps to take and I have decided to start with the conditional AutoEncoder. I have been reading some papers and I have found some interesting ideas that would be nice to implement.
+
+While stuck in the Munich airport, I started to write some code for this (the weather was not very good and my flight was delayed, so I lost my connecting flight and I had to sleep in the airport). I found an implementation of a regression variational AutoEncoder `in this paper <https://doi.org/10.1007/978-3-030-32245-8_91>`_, where the authors implement a way to manipulate the latent space so that the samples (streamlines) that get projected into it are organized along a desired scalar parameter.
```

Comment:
Minor detail, but it is "stuck at the (...)"

Reply:
sleep at the airport

Reply:
Thanks for the grammar check :)